Documentation/helm.md

```diff
@@ -3,12 +3,16 @@ title: Helm Charts
 weight: 10000
 ---
 
+{% include_relative branch.liquid %}
+
 # Helm Charts
 
-Rook has published a Helm chart for the [operator](helm-operator.md). Other Helm charts will also be potentially developed for each of the
-CRDs for all Rook storage backends.
+Rook has published the following Helm charts for the Ceph storage provider:
 
-* [Rook Ceph Operator](helm-operator.md): Installs the Ceph Operator
+* [Rook Ceph Operator](helm-operator.md): Configures the Ceph Operator
 * [Rook Ceph Cluster](helm-ceph-cluster.md): Configures resources necessary to run a Ceph cluster
 
-Contributions are welcome to create our other Helm charts!
```

Comment:
Contributions are still of course welcome, but they are welcome for all features, so it seemed redundant!